kci_build

```diff
@@ -395,7 +395,7 @@ class cmd_build_kernel(Command):
 
 
 class cmd_install_kernel(Command):
-    help = "Install the kernel binaries and build.json locally"
+    help = "Install the kernel binaries and bmeta.json locally"
```

Comment:
@gctucker which is the semantic behind `bmeta` ? Is it just `binary metadata` ?

Reply:
It's the kernel build meta-data.